hr_personal_equipment/views/hr_personal_equipment.xml

```diff
@@ -0,0 +1,116 @@
+                        <field name="product_uom_id" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
+                    </group>
+                    <group>
+                        <field name="equipment_request_id" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
+                    </group>
+                    <group name="date">
+                        <field name="start_date" attrs="{'readonly': [('state', 'not in', ('draft','accepted'))]}"/>
+                    </group>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers"/>
+                    <field name="activity_ids" widget="mail_activity"/>
+                    <field name="message_ids" widget="mail_thread"/>
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="hr_personal_equipment_tree_view">
+        <field name="name">hr.personal.equipment.tree (in hr_personal_equipment)</field>
+        <field name="model">hr.personal.equipment</field>
+        <field name="arch" type="xml">
+            <tree string="Employee Personal Equipments" create="0">
+                <field name="name" invisible="1"/>
+                <field name="product_id"/>
+                <field name="employee_id" groups="hr.group_hr_user"/>
+                <field name="quantity"/>
+                <field name="product_uom_id"/>
+                <field name="start_date"/>
+                <field name="state"/>
+                <field name="equipment_request_id" invisible="1" />
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="hr_personal_equipment_search_view">
+        <field name="name">hr.personal.equipment.search (in hr_personal_equipment)</field>
+        <field name="model">hr.personal.equipment</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="state"/>
+                <field name="employee_id" groups="hr.group_hr_user"/>
+                <field name="product_id"/>
+                <group expand="0" name="group_by" string="Group By">
+                    <filter name="state" string="State" context="{'group_by' : 'state'}" />
+                    <filter name="product_id" string="Product" context="{'group_by' : 'product_id'}" />
+                    <filter name="employee_id" string="Employee" context="{'group_by' : 'employee_id'}" />
+                </group>
+                <filter string="In Draft" name="draft_state" domain="[('state', '=', 'draft')]"/>
+                <filter string="Accepted" name="accepted_state" domain="[('state', '=', 'accepted')]"/>
+                 <filter string="Valid" name="valid_state" domain="[('state', '=', 'valid')]"/>
+                <filter string="Expired" name="expired_state" domain="[('state', '=', 'expired')]"/>
```

Comment:
Please never use a name for a filter that already exists as a field or it will collide when set from a context using `default_`.

My advice, always add `filter_` as a prefix or `groupby_`:
```
<group expand="0" name="group_by" string="Group By">
  <filter name="groupby_product_id" string="Product" context="{'group_by' : 'product_id'}" />
</group>
<filter string="In Draft" name="filter_draft_state" domain="[('state', '=', 'draft')]"/>
```

Reply:
Done! Thank you for your comment!